core/templates/pages/exploration-player-page/services/current-interaction.service.ts

```diff
@@ -54,9 +54,9 @@ export class CurrentInteractionService {
     private playerPositionService: PlayerPositionService,
     private playerTranscriptService: PlayerTranscriptService) {}
 
-  private static submitAnswerFn: SubmitAnswerFn = null;
-  private static onSubmitFn: OnSubmitFn = null;
-  private static validityCheckFn: ValidityCheckFn = null;
+  private static submitAnswerFn: SubmitAnswerFn | null;
```

Comment:
Why can this be `null`?

Reply:
The following functions will be null when the interaction passes the null value. Added the comments for the same.